packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -88,7 +100,17 @@ const GridToolbarExport = React.forwardRef<HTMLButtonElement, GridToolbarExportP
       }
     };
 
-    if (csvOptions?.disableToolbarButton && printOptions?.disableToolbarButton) {
+    const allowsExcelExport =
+      rootProps.signature === GridSignature.DataGridPro &&
+      (rootProps as DataGridProProps)?.experimentalFeatures?.excelExport;
```

Comment:
I'm not sure if we need to hide that behind the `experimentalFeatures` flag, if there are issues with it it won't break the usability of the grid. If, on the other hand, we agree that bigger features need to always be released under that flag initially then we need to rethink the features we currently develop as well. 

Tagging @m4theushw for some feedback on this.

Reply:
It's mostly to avoid breaking changes when releasing the premium plan